virttest/env_process.py

```diff
@@ -409,6 +409,17 @@ def postprocess_vm(test, params, env, name):
         except Exception:
             pass
 
+    if params.get("vm_extra_dump_paths") is not None:
+        vm_extra_dumps = os.path.join(test.outputdir, "vm_extra_dumps")
+        if not os.path.exists(vm_extra_dumps):
+            os.makedirs(vm_extra_dumps)
+        for dump_path in params.get("extra_dump_paths").split(";"):
```

Comment:
Seems you forgot about the change for this one.

Reply:
Yup sorry - updated again